src/assets/server/rest/v1/schemas/transaction/transactions-get.json

```diff
@@ -38,16 +38,16 @@
       "sanitize": "mongo"
     },
     "ConnectorID": {
-      "$ref": "chargingstation.json#/definitions/connectorIDs"
+      "$ref": "common.json#/definitions/string-ids"
```

Comment:
number-ids ?

Reply:
It was created as string for the filter (tried with numbers but it failed):

```
"connectorIDs": {
      "type": "string",
      "pattern": "^(\\d\\|?)*$",
      "sanitize": "mongo"
    },
```

Reply:
Now that number-ids is fixed, it should work

Reply:
Yes indeed :-).
Just fixed it.